ubuntudesign/design.ubuntu.com
# Reimplement download filtering

Filtering wasn't working. It now, hopefully, does.

QA
--

Run the site:

``` bash
bundle install
bundle exec jekyll serve
```

Browse to <http://127.0.0.1:4000/downloads>. Try ticking the boxes to filter. Compare it to <http://design.ubuntu.com/downloads>. See that is is different.
Be fine with it.
